devtools/db_import/db_import/db_test.py

```diff
@@ -0,0 +1,60 @@
+## Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import contextlib
+import unittest
+
+from google.cloud import bigquery
+
+from db_import import db
+from db_import import bigquery_emulator
+
+
+@unittest.skipUnless(bigquery_emulator.is_bigquery_emulator_available(
+), "The BigQuery emulator is not available on this machine. Check out https://github.com/goccy/bigquery-emulator for more details."
+                    )
```

Comment:
The auto-format behaves strangely here. I'm wondering if the trick below will make it better (adding a comma at the end)

```suggestion
@unittest.skipUnless(
  bigquery_emulator.is_bigquery_emulator_available(),
  "The BigQuery emulator is not available on this machine. Check out https://github.com/goccy/bigquery-emulator for more details.",
)
```

Reply:
Yes. that works. Fixed. Thx.